ollama-model-service/docs/access.md

```diff
@@ -0,0 +1,107 @@
+# Access & Usage Information
+
+## Basic Information
+
+### Model Server URL
+
+The model server API can be accessed at: https://ollama-route-ollama.apps.domain.com/v1
+
+## Authentication
+
+The model server does not require authentication to access, and can be accessed over a Kubernetes service or an external URL.
+
+**Note:** If you are unable to access the model server via SSO you need to reach out to a platform administrator.
+
+## API Schema
+<!--
+The name of the api, ollama-service-api, is grabbed from the name field in the ai-catalog.yaml metadata for the api.
+
+TechDocs read this as a URL starting from where the docs are located, so we can use ../ behaviour to navigate the TechDocs to reference other resources/components/apis
+-->
+The API Schema is available [here](../../../../../api/ollama-service-api/definition).
```

Comment:
This link directs me to `https://backstage-developer-hub-developer-hub.apps.rosa.redhat-ai-dev.m6no.p3.openshiftapps.com/catalog/api/ollama-service-api/definition` instead of `https://backstage-developer-hub-developer-hub.apps.rosa.redhat-ai-dev.m6no.p3.openshiftapps.com/catalog/default/api/ollama-service-api/definition`. I know it's really annoying and probably has to do with the directory structure so it won't match 1-1 with the initial catalog for how to navigate the link

Reply:
Switching to absolute paths seemed to fix this